docs/thirdparty.rst

```diff
@@ -55,14 +55,14 @@ LiveProxy
 Wtwitch
 -------
 
-.. image:: https://i.imgur.com/oaxH3ss.jpg
+.. image:: https://git.sr.ht/~krathalan/wtwitch/blob/refs/heads/master/screenshot.jpg
```

Comment:
That's not a static image URL. If you have an image file in the git tree (which btw is always a bad idea), at least link to a static commit ID.

Reply:
1. use a github url, not a third party domain

example https://github.com/streamlink/streamlink/blame/master/docs/applications.rst#L11

2. compress the image, your image is 730kB vs 135kB from the streamlink twitch gui